### PR TITLE
Feature/feature profile picture

### PR DESCRIPTION
**Pull request by xavier-campos-diaz**

### CODE REVIEW

File: app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/ImageZoom.java [hunk 1]
```diff
@@ -111,4 +113,12 @@ public void setDrawable(Drawable drawable) {
         ImageZoom.drawable = drawable;
         initializeCircularImageView();
     }
+
+    public static boolean isMainActivity() {
```
Comment (AlbertPG) [inline app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/ImageZoom.java:121]: You should comment this method and the next one

File: app/src/main/java/org/pesmypetcare/mypetcare/features/users/User.java [hunk 2]
```diff
@@ -64,6 +67,14 @@ public void deletePet(Pet pet) {
         return pets;
     }
 
+    public Bitmap getUserProfileImage() {
```
Comment (AlbertPG) [inline app/src/main/java/org/pesmypetcare/mypetcare/features/users/User.java:74]: You should comment this methods as well

File: app/src/main/res/values/styles.xml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <resources>
-    <style name="AppTheme" parent="Theme.MaterialComponents.Light.NoActionBar">
```
Comment (AlbertPG) [inline app/src/main/res/values/styles.xml]: You should keep the NoActionBar

**Review by AlbertPG — COMMENTED** [review]
You should take into account thiese issues

**Review by enrichp — APPROVED** [review]

**Review by AlbertPG — APPROVED** [review]
Okey, you've got my approval

**Pull request merged**